README.md

```diff
@@ -1058,6 +1058,46 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='omit-single-expression-return'></a>(<a href='#omit-single-expression-return'>link</a>) **Omit the `return` return keyword from single-expression functions accessors, and closures.** [![SwiftLint: omit-single-expression-return](https://img.shields.io/badge/SwiftLint-implicit-return-007A87.svg)](https://realm.github.io/SwiftLint/implicit_return.html)
+
+<details>
+
+```swift
+// WRONG
+["1", "2", "3"].compactMap { return Int($0) }
+
+var size: CGSize {
+  return CGSize(
+    width: 100.0,
+    height: 100.0)
+}
+
+func buildInfoAlert(message: String) -> UIAlertController {
```

Comment:
nit: the Swift API design guidelines specify that:
> Begin names of factory methods with “make”, e.g. x.makeIterator().

https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage

So this would be `makeInfoAlert`